Plugins/Source/KISAddonCursor.cs

```diff
@@ -89,6 +82,17 @@ sealed class KISAddonCursor : MonoBehaviour {
     Cursor.visible = false;
   }
 
+  void Awake() {
+    hintWindowStyle = new GUIStyle {
```

Comment:
This is awkward. It's a static member, it should get initialized before anything else is triggered. If it doesn't (e.g. due to KSP cloning) then member must be turned into instance member.